react/features/analytics/functions.js

```diff
@@ -43,10 +43,15 @@ export function initAnalytics({ getState }: { getState: Function }) {
 
     const state = getState();
     const config = state['features/base/config'];
-    const { analyticsScriptUrls, deploymentInfo, googleAnalyticsTrackingId }
-        = config;
+    const {
+        amplitudeAPPKey,
```

Comment:
Can we put this in some `analytics` key instead of all of them at the top level?

Reply:
Maybe just without the deploymentInfo, don't we have too much stuff reading that, not only analytics but infra and other features?

Reply:
Sure. Whatever little progress we make in putting things in the right place is still (some) progress!

Reply:
@saghul @damencho Do you want me to put only `amplitudeAPPKey` into `analytics` or everything without `deploymentInfo`?

If you want me to put everything without `deploymentInfo` into `analytics` key aren't we going to break the backward compatibility? This is my only concern otherwise I was also tempted to do it!

Reply:
I'd rather have it on a top-level `analytics` key. You can add some code to read from the old location (just the old keys) as a transitional meassure.